anabot/runtime/installation/hub/connect_to_redhat/registration.py

```diff
@@ -38,6 +38,12 @@ def registration_info_panel(local_node):
     sibling_filler = getparent(unreg_button, 'filler')
     return getsibling(sibling_filler, 1, "panel")
 
+handle_act('/account_organization', default_handler)
```

Comment:
I believe that the "account_" can be dropped from the name as there should be no other "organization" related information in the information about the subscription/registration so the element path could simply be:
/installation/hub/connect_to_redhat/registration/organization

@jstodola what do you think?

Reply:
@pholica I named it this way so that it would be consistent with the other functions that pertain to the combobox. But if you think it should be changed, I will change it.